docs/general/resources/res_desc.json

```diff
@@ -3244,10 +3244,12 @@
         "note": ""
     },
     "501": {
-        "name": "Basileus Dead",
-        "desc": "Unknown... What does this resource do?",
-        "defaults": {},
-        "note": "Incremented when `Basileus Dead` unit is spawned."
+        "name": "Polemarch Resource",
```

Comment:
Since the game doesn't give this resource a name, we can call it something more mnemonic, "Maximum Polemarchs" perhaps? Since that's what it controls